docs/latest/tutorial/tutorial-main-renderer.md

```diff
@@ -0,0 +1,189 @@
+---
+title: 'Main and Renderer Process Communication'
```

Comment:
I don't think this title is super accurate because the preload script doesn't actually communicate with the main process. There's no IPC involved here AFAIK, the preload script is just privileged and has access to some Node.js APIs that the renderer process does.

I'd maybe rename this to `Using a preload script` or something similar.

Reply:
You are right.
What about adding a small IPC section right after. Could be in another document or this same one.

Reply:
This code review reminded me to pick up writing the IPC doc again. Maybe add a TODO for me?

Reply:
I've added a todo at the end of the doc.
Still need to think about a better name 🤔
What about this?

> Exposing features in the renderer process

Reply:
Maybe "Using Electron and Node APIs in the renderer" would be a bit more descriptive, not sure if it's too wordy.

Reply:
```suggestion
title: 'Using Electron and Node APIs in the renderer'
```

Sold